packages/at_secondary_server/test/enroll_verb_test.dart

```diff
@@ -1154,13 +1155,13 @@ void main() {
       String unrevokeEnrollmentCommand = 'enroll:unrevoke:{"enrollmentId":""}';
       enrollVerbParams =
           getVerbParam(VerbSyntax.enroll, unrevokeEnrollmentCommand);
-      await expectLater(
+      expect(
```

Comment:
Why has this been changed?

Reply:
expectLater caused the unit tests to fail in [failed GitHub actions run](https://github.com/atsign-foundation/at_server/actions/runs/10929335864/job/30339826816). Changed it to 'expect' to ensure the tests pass.